modules/ols-openshift-lightspeed-fips-support.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+// * lightspeed-docs-main/configure/ols-configuring-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="openshift-lightspeed-fips-support_{context}"]
+= {ols-long} FIPS support
+
+{ols-official} is designed for Federal Information Processing Standards (FIPS).
```

Comment:
This sentence is awkward to me. 
```suggestion
{ols-official} is designed to meet the Federal Information Processing Standards (FIPS).
```
```suggestion
{ols-official} meets the Federal Information Processing Standards (FIPS).
```

Reply:
As I understand it, this sentence was massaged with a team, including legal, and essentially it was important to say designed for rather than meets because the approval process is lengthy and complicated. I think I might leave this part as is, but I did make the other changes you suggested. Thanks!

Reply:
Good to know!